mongo/errors.go

```diff
@@ -162,28 +161,34 @@ func writeErrorsFromDriverWriteErrors(errs driver.WriteErrors) WriteErrors {
 	return wes
 }
 
-// WriteConcernError is a write concern failure that occurred as a result of a
-// write operation.
+// WriteConcernError represents a write concern failure during execution of a write operation. This error type is only
+// returned as part of a WriteException or a BulkWriteException.
 type WriteConcernError struct {
 	Name    string
 	Code    int
 	Message string
-	Details bson.Raw
+	Details bson.Raw // A document identifying the write concern setting related to the error.
```

Comment:
Not sure what you mean by this.  What's an example of a details document?

Reply:
I actually don't know. It's documented in https://github.com/mongodb/specifications/blob/master/source/crud/crud.rst but isn't mentioned in the linked page (https://docs.mongodb.com/manual/reference/method/WriteResult/).

Reply:
From what I recall, the details document does not refer to the setting per se, but more particulars about the error that occurred.  Either research what's actually in there, or make the doc string more generic (or just remove it)

Reply:
I removed this. It doesn't seem very useful and isn't consistently returned by the server. If we decide it's helpful in the future, we can probably make this something other than a `bson.Raw` or add helpers to make it usable.